### ISSUE TITLE
After chapter 8: We can still post duplicates of one URL with the Edit form

### ISSUE THREAD

**Issue by Yaakov-Belch** [DiscoverMeteor/book#330]
Microscope tries to prevent multi-submissions of the same URL.  But it can still be done:
- Submit your post with a fake, random URL.
- Edit the post to point to the non-unique URL.

As a proof of concept, I added three posts for [themeteorbook.com](http://themeteorbook.com/) to your instance [meteor-book-chapter8-3.meteor.com](http://meteor-book-chapter8-3.meteor.com).

To fix this problem, you would need to copy the de-duplication code from the post submission to the edit feature --- convert the client-side data base change to a meteor Method.

**Comment by pors** [comment]
I agree, I don't understand why editng posts is done differently from submitting posts.
